configs/train_tapecam_simulated_background.yaml

```diff
@@ -0,0 +1,54 @@
+# python scripts/recon/train_learning_based.py -cn train_mirflickr_tape
+defaults:
+  - train_unrolledADMM
+  - _self_
+
+torch_device: 'cuda:0'
+#device_ids: [0, 1, 2, 3]
+eval_disp_idx: [ 1, 2, 4, 5, 9 ]
+
+wandb_project:
+
+# Dataset
+files:
+  dataset: bezzam/TapeCam-Mirflickr-25K
+  huggingface_dataset: True
+  huggingface_psf: psf.png
+  downsample: 1
+  # TODO: these parameters should be in the dataset?
+  image_res: [ 900, 1200 ]  # used during measurement
+  rotate: False   # if measurement is upside-down
+  save_psf: True
+
+  background_fp: ""
+  background_snr_range: [ -10, 10 ]
```

Comment:
by inhering from `train_mirflickr_tape` only these two lines need to be specified